Zend/zend_alloc.c

```diff
@@ -839,8 +872,8 @@ static void *zend_mm_chunk_alloc_int(size_t size, size_t alignment)
 static void *zend_mm_chunk_alloc(zend_mm_heap *heap, size_t size, size_t alignment)
 {
 #if ZEND_MM_STORAGE
-	if (UNEXPECTED(heap->storage)) {
-		void *ptr = heap->storage->handlers.chunk_alloc(heap->storage, size, alignment);
+	if (UNEXPECTED(ZEND_MM_RO_HEAP(heap)->storage)) {
```

Comment:
Nit pick: We could use two macros to access `->storage` and `->custom_heap`. This would reduce verbosity. WDYT?

```
#define ZEND_MM_STORAGE(heap)     ZEND_MM_RO_HEAP(heap)->storage
#define ZEND_MM_CUSTOM_HEAP(heap) ZEND_MM_RO_HEAP(heap)->custom_heap
```

Reply:
I'd rather not to, to make it explicit that `storage` and `custom_heap` are both stored in the ro-heap.